src/universal/components/OutcomeCard/OutcomeCardFooter.js

```diff
@@ -114,7 +120,7 @@ const OutcomeCardFooter = (props) => {
       </div>
       <div className={styles.buttonBlock}>
         <button className={buttonStyles} onClick={handleStatusClick}>
```

Comment:
Just gave all these beautiful enhancements a try – everything worked _except_ I think this `handleStatusClick` handler is still wired up to have the user switch project owners, not switch the project's team.